### ISSUE TITLE
nonenglish comments

### ISSUE THREAD

**Issue by matkoniecz** [opening-hours/opening_hours_map#51]
for example in main.js there are:

https://github.com/opening-hours/opening_hours_map/blob/master/js/main.js#L520
https://github.com/opening-hours/opening_hours_map/blob/master/js/main.js#L299
https://github.com/opening-hours/opening_hours_map/blob/master/js/main.js#L287
https://github.com/opening-hours/opening_hours_map/blob/master/js/main.js#L280
https://github.com/opening-hours/opening_hours_map/blob/master/js/main.js#L130
https://github.com/opening-hours/opening_hours_map/blob/master/js/main.js#L114
https://github.com/opening-hours/opening_hours_map/blob/master/js/main.js#L109

**Comment by ypid** [comment]
Yup, this code is still as Netztwolf wrote it. No worry, I try to maintain all my projects in English. Will need to be fixed in a rework.
